menu.js

```diff
@@ -44,6 +44,14 @@ export function menuKeyUX() {
       } else if (event.key === 'End') {
         event.preventDefault()
         focus(event.target, items[items.length - 1])
+      } else {
```

Comment:
We need not only to find by first letter, but also by some pattern of letters pressed with a small delay.

Like:

1. `a` — looking for `a`
2. `b` after 100ms — looking for `ab`
3. `c` after 50ms — looking for `abc`
4. `d` after 5000ms — looking for `d`